src/commands/game_commands/lookup.ts

```diff
@@ -39,11 +44,290 @@ const getDaisukiLink = (id: string, isMV: boolean): string => {
+
+        const artistNameQuery = await dbContext
+            .kpopVideos("app_kpop_group")
+            .select("name", "kname")
+            .where("id", daisukiSongEntry.id_artist)
+            .first();
+
+        artistName =
+            artistNameQuery.kname && isKorean
+                ? artistNameQuery.kname
+                : artistNameQuery.name;
+
+        songAliases = [...daisukiSongEntry.name_aka.split(";")].join(", ");
+        songAliases += songAliases
+            ? `, ${daisukiSongEntry.kname}`
+            : daisukiSongEntry.kname;
+
+        artistAliases = State.aliases.artist[artistNameQuery.name]?.join(", ");
+
+        views = daisukiSongEntry.views;
+        publishDate = new Date(daisukiSongEntry.publishedon);
+
+        logger.info(
+            `${getDebugLogHeader(
+                message
+            )} | Non-KMQ song lookup. videoID = ${videoID}.`
+        );
+    }
+
+    const viewsString = LocalizationManager.localizer.translate(
+        guildID,
+        "misc.views"
+    );
+
+    const fields = [
+        {
+            name: viewsString[0].toUpperCase() + viewsString.slice(1),
+            value: friendlyFormattedNumber(views),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.releaseDate"
+            ),
+            value: friendlyFormattedDate(publishDate, guildID),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.songAliases"
+            ),
+            value:
+                songAliases ||
+                LocalizationManager.localizer.translate(guildID, "misc.none"),
+        },
+        {
+            name: LocalizationManager.localizer.translate(
+                guildID,
+                "misc.artistAliases"
+            ),
+            value:
+                artistAliases ||
+                LocalizationManager.localizer.translate(guildID, "misc.none"),
+        },
+    ];
+
+    if (kmqSongEntry) {
+        fields.push(
+            {
+                name: LocalizationManager.localizer.translate(
+                    guildID,
+                    "misc.duration"
+                ),
+                value:
+                    songDuration ||
+                    LocalizationManager.localizer.translate(
+                        guildID,
+                        "misc.notApplicable"
+                    ),
+            },
+            {
+                name: LocalizationManager.localizer.translate(
+                    guildID,
+                    "command.lookup.inCurrentGameOptions"
+                ),
+                value: LocalizationManager.localizer.translate(
+                    guildID,
+                    includedInOptions ? "misc.yes" : "misc.no"
+                ),
+            }
+        );
+    }
+
+    sendInfoMessage(messageContext, {
+        title: `${songName} - ${artistName}`,
```

Comment:
Nit: The song name is not bold here but is bold in the results, which works better?